tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_new_window.py

```diff
@@ -0,0 +1,56 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By, Wait
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.search.app import Search
+from gaiatest.apps.system.regions.cards_view import CardsView
+
+
+class TestBrowserNewWindow(GaiaTestCase):
```

Comment:
I'd rename this class and the file to something like: `test_browser_cards_in_cards_views`